app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -3410,12 +3412,15 @@ private void updateStreamRelatedViews() {
                 break;
 
             case VIDEO_STREAM:
-                if (info.getVideoStreams().size() + info.getVideoOnlyStreams().size() == 0) {
+                if (currentMetadata == null
+                        || !currentMetadata.getMaybeQuality().isPresent()
+                        || info.getVideoStreams().size() + info.getVideoOnlyStreams().size() == 0) {
```

Comment:
```suggestion
                        || (info.getVideoStreams().isEmpty() && info.getVideoOnlyStreams().isEmpty())) {
```

I would write it like this - better understandable IMHO